build/build_conda.sh

```
@@ -385,10 +377,10 @@ main() {
     # Execute selected build steps
     if [ "$DOWNLOAD_PIPELINE" = true ]; then
         download_pipeline
-    fi
-    
-    if [ "$DOWNLOAD_CONTAINERS" = true ]; then
         download_containers
+	# Pack all cloned repositories
+        echo "Creating pipeline archive: ${PIPELINE_NAME}_${TAG_OR_BRANCH}.tar.gz"
+        tar -zcvf ${PIPELINE_NAME}_${TAG_OR_BRANCH}.tar.gz ${PIPELINE_NAME}_${TAG_OR_BRANCH}
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)